### DISCUSSION TITLE
Could not find a declaration file for module './presets/aura'.

### DISCUSSION THREAD

**Discussion by harunaabaldeh** [primefaces/community#2752] [PrimeVue]
I have followed the steps mentioned [here](https://tailwind.primevue.org/vite/) but I still have the error `Could not find a declaration file for module './presets/aura'. `

I am using vue, tailwindcss, and typescript. The error occurred in my `main.ts` file.

**Comment by qwadrox** [comment]
Hi @harunaabaldeh. This error indicates that TypeScript cannot find the type definitions for the JavaScript module you're trying to import.  You might need to [tweak tsconfig.json](https://www.typescriptlang.org/tsconfig/#allowJs): 
```ts
{
  "compilerOptions": {
    "allowJs": true
  }
}
```
Or create a typescript declaration file: aura.d.ts
